tools/jit/gen_jit_dispatch.py

```diff
@@ -91,20 +99,24 @@ def gen_jit_dispatch(declarations, out):
     def is_tensor_arg(arg):
         return arg['simple_type'] in {'Tensor', 'TensorList'}
 
+    def is_sized_intlist_arg(arg):
```

Comment:
Would be nice to add a comment because it's not obvious what a "sized intlist" is. Can you add a docstring saying sth like
```
"""Returns true for args declared as IntList[x], but not IntList."""
```